File: Python/BruteForce.py
def isPrime(num):
    if num <= 1:
        return False
    elif num==2:
        return True
    for i in range(2, num):
        if num%i == 0:
            return False
    return True

test = int(input("Enter a number to check if it's prime or composite: "))
print("The number is "+("prime" if isPrime(test) else "not prime"))

File: Java/BruteForce.java
import java.util.Scanner;

public class Main {
    public static boolean isPrime(int num) {
        if (num <= 1) return false;
        if (num == 2) return true;
        for (int i = 2; i<num; i++)
            if (num%i==0)
                return false;
        return true;
    }

    public static void main(String[] args) {
        System.out.println("Enter a number to check if it's prime or composite");
        Scanner input = new Scanner(System.in);
        int num = input.nextInt();
        System.out.println("The number you entered is "+(isPrime(num)?"prime":"not prime"));
    }
}
